app/decorators/ext_management_system_decorator.rb

```diff
@@ -6,6 +6,10 @@ def fonticon
   end
 
   def listicon_image
-    "100/vendor-#{image_name}.png"
+    if kind_of?(ManageIQ::Providers::Hawkular::MiddlewareManager)
+      'vendor-hawkular'
+    else
+      "100/vendor-#{image_name}.png"
```

Comment:
This doesn't look much like `'vendor-hawkular'`; I have no opinion on whether this method should return a short fragment or the full filename, but it seems likely it should consistently do one or the other.